code/WorkInProgress/AbilityItem.dm

```diff
@@ -854,7 +854,8 @@
 	layer = HUD_LAYER
 	plane = PLANE_HUD
 	anchored = ANCHORED
-	flags = NOSPLASH
+	flags = 0
```

Comment:
```suggestion
```
I'd just remove the `flags` overrides from where you remove the NOSPLASH. I know that this is technically changing the behaviour as default flags are FPRINT (a mistake imo, we should have had a flag for NO_FPRINT instead) but I guarantee you that 99% of cases of removing the FPRINT flag from things are accidental.

Reply:
gotcha